src/createSession.js

```diff
@@ -1,3 +1,35 @@
+import axios from 'axios';
+
+const libraryPackage = require('../package.json');
+
+const logAnalyticsEvent = (apiKey, sessionId, action, connectionId) => {
+  const body = {
+    payload: {
+      opentok_react_version: libraryPackage.version,
+    },
+    payload_type: 'info',
+    action,
+    partner_id: apiKey,
```

Comment:
could we move this to camelCase? `partnerId` for example?

Reply:
Yes, will fix. I have old notes from Mike, but I'll update to use the current schema